topgrade.8

```diff
@@ -2,14 +2,14 @@
 .\" Contact github/Linux4Life531 to correct errors or typos.
 .TH man 8 "22 Febuary 2020" "1.0" " Topgrade Manpage"
 .SH NAME
-topgrade \- upgrade everything on your computer
+topgrade - upgrade everything on your computer
 .SH SYNOPSIS
 topgrade
 .SH DESCRIPTION
 Keeping your system up to date mostly involves invoking more than a single package manager. This usually results in big shell one-liners saved in your shell history. Topgrade tries to solve this problem by detecting which tools you use and run their appropriate package managers.
 .SH OPTIONS
 The topgrade does not take any options.
```

Comment:
There are arguments. Please run `topgrade --help` to see them

Reply:
Okay.

Reply:
Okay.

Reply:
I think I have added them in. Thanks